docs/resources/rest_managed.md

```diff
@@ -191,7 +191,7 @@ Importing an object using its distinguished name (DN) with a JSON string:
 
 ```
 import {
-  id = `{ "parentDn": "uni/tn-{name}" }`
+  id = "{\"parentDn\": \"uni/tn-{name}\"}"
```

Comment:
Shall we add an explanation stating the reason for escaping double quotes? Something like the outer quotes define the string boundary in HCL due to which the parser would fail without escaping?